credentialz/credentialz.proto

```diff
@@ -677,7 +677,8 @@ message CaPublicKeyResponse {
 message AuthorizedPrincipalCheckRequest {
   enum Tool {
     TOOL_UNSPECIFIED = 0;
-    TOOL_HIBA_DEFAULT = 1;
+    TOOL_NOT_CONFIGURED = 1;
```

Comment:
why are you suggesting adding this enum?
what is different from not_configured vs unspecified?
Also what if not hiba would be used for principal check?

Reply:
Per the protobuf style guide (https://protobuf.dev/programming-guides/style/#enums) UNSPECIFIED seems like a sentinel value to be used as a default, but it does not indicate an explicit configuration request. NOT_CONFIGURED is used to explicitly request that hiba-chk not be used.

Reply:
If HIBA is not used we can use authorized principals file for principal check, it can be enabled by
1. `RotateAccountCredentialsRequest{AuthorizedUsersRequest}` to rotate principals for users.
2. `RotateHostParametersRequest{CaPublicKeyRequest}`, this rotates CA public keys and also enables authorized principals file for all users.

Reply:
unspecified is defined as the default - it is up the api design to decide what to do with a default i.e. it is a "valid value" but does require protocol specification on what to do with it

Reply:
you should not redefine tags - just add the new entry as tag 2

Reply:
We have removed the proposed new enum and added documentation that `TOOL_UNSPECIFIED` removes any previously configured authorized principals command from configuration.